### ISSUE TITLE
Ability to choose which dashboards to import for Kibana

### ISSUE THREAD

**Issue by radoondas** [elastic/packetbeat-dashboards#89]
Use case here is that if I run only topbeat, I don't want to import all other beats in to my Kibana.
I think it can be taken in consideration in next generation of dashboards when (for example) merged to beats itself. 

I have a little idea how to do it with current setup. it would require only little changes in to the structure of dashboards and load script itself. If there is interest for the enhancement, I can prepare PR.

Is is worth to do it? Or am I the only one with 'issue'.

**Comment by ruflin** [comment]
I plan to solve this by moving the dashboards to each beat: https://github.com/elastic/beats/issues/1102 See https://github.com/elastic/beats/pull/1103 as a first approach.

**Comment by radoondas** [comment]
OK, this makes sense and I like it. 
I'll leave this as is and maybe help with elastic/beats#1102.

**Comment by ghost** [comment]
What's the current status of this work?  I'm having trouble following the discussions on links to other issues that seemed to be closed.  We also only run topbeat and would like a way to load only that one into Kibana.
Thanks.

**Comment by damm** [comment]
Yeah it would; reduces breakage when people load the dashboards because they are not loading so many components.  

really beats-dashboards should be broken down into the sub components that it's doing vis for; you could load the topbeat component and render topbeat and be done.

Only load what you need to survive.

**Comment by monicasarbu** [comment]
The beats-dashboards repository is not used anymore. The dashboard(s) are spread to each Beat repository (see [etc/kibana](https://github.com/elastic/beats/tree/master/packetbeat/etc/kibana) directory). You can now import only the dashboard(s) of a single Beat by using the `import_dashboards.sh` script available in the Beat package and also [here](https://github.com/elastic/beats/blob/master/dev-tools/import_dashboards.sh).

**Comment by monicasarbu** [comment]
Closing this issue as it is fixed using the `import_dashboards.sh` available in the beats/dev-tools repository.

**Comment by ghost** [comment]
@monicasarbu: This doesn't look to be the case if we're using topbeat.  1.2.3 looks like the latest version and it's not packaged with the dashboards.  I know metricbeat is replacing it, but that's still in alpha for ES 5.0.  I'm using ES 2.3.3.

**Comment by ghost** [comment]
Can anyone tell me which searches and visualizations in the beats-dashboards package go with topbeats.  I think if I can just delete the others from the dashboards directory maybe I can use the import_dasboards.sh script to load just the json files for topbeats.

Thanks.

**Comment by damm** [comment]
@PMDubuc https://github.com/damm/beats-dashboards/commit/66c8ba55727409f44b3aba65a3355a3af0e33cc7

That's what I did awhile back... it may not be all inclusive

**Comment by ghost** [comment]
@damm Thank you!

**Comment by monicasarbu** [comment]
@PMDubuc Yes, you are right. Unfortunately it's not available in 1.2.3 and it would be a bit difficult to backport the changes from 5.0. The good thing is that soon 5.0 will be GA.

**Comment by ghost** [comment]
This works: Copy the dashboards directory out of the beats-dashboards-1.2.3.zip to a topbeat-dashboards directory and remove all but the following:

```
ls -R topbeat-dashboards/
topbeat-dashboards/:
dashboard  index-pattern  search  visualization

topbeat-dashboards/dashboard:
Topbeat-Dashboard.json

topbeat-dashboards/index-pattern:
topbeat.json

topbeat-dashboards/search:
Default-Search.json    Processes.json   System-stats.json
Filesystem-stats.json  Proc-stats.json  System-wide.json

topbeat-dashboards/visualization:
Average-system-load-across-all-systems.json
CPU-usage.json
CPU-usage-per-process.json
Disk-usage.json
Disk-usage-overview.json
Disk-utilization-over-time.json
Evolution-of-the-CPU-times-per-process.json
Memory-usage.json
Memory-usage-per-process.json
Process-status.json
Servers.json
System-load.json
Top-10-memory-consumers.json
Top-10-processes-by-total-CPU-usage.json
Top-processes.json
```

Then you can use the import_dashboards.sh script like this:

./import_dashboards.sh -dir ./topbeat-dashboards -url http://${HOST}:${PORT}

When you display the Topbeat-Dashboard you can delete the empty Navigation pane and save (overwrite) the Topbeat-Dashboard.
